### PR TITLE
Updates copy on /openstack/compare

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates copy on /openstack/compare

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/compare
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1ibpATi-e0rKP3GaLgvzwO_XyFU9ZTsUaCHMwrg96MZM/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10043

### CODE REVIEW

File: templates/openstack/compare.html [hunk 1]
```diff
@@ -308,7 +308,7 @@ <h1>OpenStack distributions: <br class="u-hide--small">a comparison</h1>
     <div class="col-8">
       <h2>More economical than Red Hat OpenStack Platform</h2>
       <p>By applying a per host pricing model to the subscription structure and offering design and delivery at a fixed price, Canonical's Charmed OpenStack is cheaper than Red Hat OpenStack Platform. Moreover, no license is required to get started with Charmed OpenStack. This means that organisations only have to pay for the environments that require enterprise support, such as the production environment, while they can run their testing and development environments free of charge.</p>
-      <p><a href="/engage/redhat-openstack-comparison-whitepaper">Learn more &nbsp;&rsaquo;</a></p>
+      <p><a href="/engage/redhat-openstack-comparison-whitepaper">Read a Red Hat OpenStack comparison whitepaper &nbsp;&rsaquo;</a></p>
```
Comment (bethcollins92) [inline templates/openstack/compare.html:311]:
```suggestion
      <p><a href="/engage/redhat-openstack-comparison-whitepaper">Read a Red Hat OpenStack comparison whitepaper&nbsp;&rsaquo;</a></p>
```

File: templates/openstack/compare.html [hunk 2]
```diff
@@ -358,7 +358,7 @@ <h2>More straightforward than Mirantis Cloud Platform</h2>
     <div class="col-8">
       <h2>Open source unlike VMware</h2>
       <p>Canonical's Charmed OpenStack is fully open source which allows it to benefit from a broader community, faster development process and wider selection of technology choices than those available in VMware's platform. Choosing an open source platform avoids vendor lock-in and significantly reduces infrastructure costs.</p>
-      <p><a href="/engage/vmware-to-charmed-openstack">Learn more &nbsp;&rsaquo;</a></p>
+      <p><a href="/engage/vmware-to-charmed-openstack">Explore VMware to OpenStack migration options &nbsp;&rsaquo;</a></p>
```
Comment (bethcollins92) [inline templates/openstack/compare.html:361]:
```suggestion
      <p><a href="/engage/vmware-to-charmed-openstack">Explore VMware to OpenStack migration options&nbsp;&rsaquo;</a></p>
```

File: templates/openstack/compare.html [hunk 3]
```diff
@@ -392,7 +392,7 @@ <h2>Open source unlike VMware</h2>
     <div class="col-8">
       <h2>More cost-efficient than public clouds</h2>
       <p>Public clouds are expensive when running workloads in the long-term and at scale. Although they are a more economical solution at the beginning of the cloud transformation process, their costs keep growing over time. Canonical's Charmed OpenStack is a cost-efficient extension to public cloud infrastructure, ensuring lower TCO per VM and enabling businesses to run their workloads where it makes the most sense from an economical point of view.</p>
-      <p><a href="/tco-calculator">Learn more&nbsp;&rsaquo;</a></p>
+      <p><a href="/tco-calculator">Estimate cost savings &nbsp;&rsaquo;</a></p>
```
Comment (bethcollins92) [inline templates/openstack/compare.html:395]:
```suggestion
      <p><a href="/tco-calculator">Estimate cost savings&nbsp;&rsaquo;</a></p>
```

**Review by bethcollins92 — APPROVED** [review]
LGTM! Just a suggestion to remove the spaces before the > for consistency

**Comment by petesfrench** [comment]
@bethcollins92 Nice catch, thanks!

**Pull request merged**